tcmur_cmd_handler.c

```diff
@@ -443,6 +443,127 @@ static int handle_flush(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	return async_handle_cmd(dev, cmd, flush_work_fn);
 }
 
+static int handle_rcr_op(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
```

Comment:
I think I might have made a mistake with the really short names like caw for compare_and_write. If you copied the really short style name used for caw/compare_and_write feel free to use more descriptive naming like:

handle_recv_copy_result or handle_recv_cpy_res or whatever you want.

I do not mind either way, so feel free to leave it if you like it.